QA_Nano.md

```diff
@@ -0,0 +1,107 @@
+# Create Nano Contract
+This test suite will execute the full flow of the [Bet blueprint](https://explorer.alpha.nano-testnet.hathor.network/blueprint/detail/3cb032600bdf7db784800e4ea911b10676fa2f67591f82bb62628c234e771595), ensuring all interface interaction is validated with its error handling cases.
+
+More about Nano Contracts on our [official docs](https://docs.hathor.network/explanations/features/nano-contracts/).
+
+# Initializing connection
+- Open the Desktop Wallet with an existing seed
+- Go to "Settings" and change its server to `https://hathorplay.nano-testnet.hathor.network/v1a/`
```

Comment:
We can now connect to `https://node1.nano-testnet.hathor.network/v1a/version`

Reply:
Updated the URL